libs/schema/src/index.ts

```diff
@@ -1,15 +1,7 @@
-export interface Schema {
-  name: string;
-  type: string;
-  description: string;
-  defaultValue?: string;
-  important?: boolean;
-  completion?: string;
-  deprecated?: string;
-  required: boolean;
-  positional: boolean;
-  enum?: string[];
-}
+import { Option } from '@angular/cli/models/interface';
+
+// tslint:disable-next-line: no-empty-interface
+export interface Schema extends Option {}
```

Comment:
Is the `Schema` just an alias now? Like `type Schema = Option`?

Reply:
Bingo. 

In a followup I want to remove the `Schema` interface all together and just use `Option` directly. I made an attempt to do it as part of the same change but it produced a lot of noise

Reply:
Ah cool :)